scripts/list_dataset_metadata_handler.py

```diff
@@ -8,28 +8,22 @@
 
 
 def list_dataset_metadata_handler(dataset_paths: Tuple[str]) -> List[dict]:
-    """
-    Lists metadata of given datasets like:
-    [
-       {
-          "domain":"AE",
-          "filename":"ae.xpt",
-          "full_path":"/Users/Aleksei_Furmenkov/PycharmProjects/cdisc-rules-engine/resources/data/ae.xpt",
-          "file_size":"38000",
-          "label":"Adverse Events",
-          "modification_date":"2020-08-21T09:14:26"
-       },
-       {
-          "domain":"EX",
-          "filename":"ex.xpt",
-          "full_path":"/Users/Aleksei_Furmenkov/PycharmProjects/cdisc-rules-engine/resources/data/ex.xpt",
-          "file_size":"78050",
-          "label":"Exposure",
-          "modification_date":"2021-09-17T09:23:22"
-       },
-       ...
-    ]
-    """
+    from core import VALIDATION_SUPPORTED_FORMATS
```

Comment:
[Imports should be done at the top of the file ](https://peps.python.org/pep-0008/#imports)